src/OpenSSL/SSL.py

```diff
@@ -250,6 +252,127 @@
+    X509_V_ERR_PATH_LENGTH_EXCEEDED = _lib.X509_V_ERR_PATH_LENGTH_EXCEEDED
+    X509_V_ERR_INVALID_PURPOSE = _lib.X509_V_ERR_INVALID_PURPOSE
+    X509_V_ERR_CERT_UNTRUSTED = _lib.X509_V_ERR_CERT_UNTRUSTED
+    X509_V_ERR_CERT_REJECTED = _lib.X509_V_ERR_CERT_REJECTED
+    X509_V_ERR_SUBJECT_ISSUER_MISMATCH = (
+        _lib.X509_V_ERR_SUBJECT_ISSUER_MISMATCH
+    )
+    X509_V_ERR_AKID_SKID_MISMATCH = _lib.X509_V_ERR_AKID_SKID_MISMATCH
+    X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH = (
+        _lib.X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH
+    )
+    X509_V_ERR_KEYUSAGE_NO_CERTSIGN = _lib.X509_V_ERR_KEYUSAGE_NO_CERTSIGN
+    X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER = (
+        _lib.X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER
+    )
+    X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION = (
+        _lib.X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION
+    )
+    X509_V_ERR_KEYUSAGE_NO_CRL_SIGN = _lib.X509_V_ERR_KEYUSAGE_NO_CRL_SIGN
+    X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION = (
+        _lib.X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION
+    )
+    X509_V_ERR_INVALID_NON_CA = _lib.X509_V_ERR_INVALID_NON_CA
+    X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED = (
+        _lib.X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED
+    )
+    X509_V_ERR_KEYUSAGE_NO_DIGITAL_SIGNATURE = (
+        _lib.X509_V_ERR_KEYUSAGE_NO_DIGITAL_SIGNATURE
+    )
+    X509_V_ERR_PROXY_CERTIFICATES_NOT_ALLOWED = (
+        _lib.X509_V_ERR_PROXY_CERTIFICATES_NOT_ALLOWED
+    )
+    X509_V_ERR_INVALID_EXTENSION = _lib.X509_V_ERR_INVALID_EXTENSION
+    X509_V_ERR_INVALID_POLICY_EXTENSION = (
+        _lib.X509_V_ERR_INVALID_POLICY_EXTENSION
+    )
+    X509_V_ERR_NO_EXPLICIT_POLICY = _lib.X509_V_ERR_NO_EXPLICIT_POLICY
+    X509_V_ERR_DIFFERENT_CRL_SCOPE = _lib.X509_V_ERR_DIFFERENT_CRL_SCOPE
+    X509_V_ERR_UNSUPPORTED_EXTENSION_FEATURE = (
+        _lib.X509_V_ERR_UNSUPPORTED_EXTENSION_FEATURE
+    )
+    X509_V_ERR_UNNESTED_RESOURCE = _lib.X509_V_ERR_UNNESTED_RESOURCE
+    X509_V_ERR_PERMITTED_VIOLATION = _lib.X509_V_ERR_PERMITTED_VIOLATION
+    X509_V_ERR_EXCLUDED_VIOLATION = _lib.X509_V_ERR_EXCLUDED_VIOLATION
+    X509_V_ERR_SUBTREE_MINMAX = _lib.X509_V_ERR_SUBTREE_MINMAX
+    X509_V_ERR_UNSUPPORTED_CONSTRAINT_TYPE = (
+        _lib.X509_V_ERR_UNSUPPORTED_CONSTRAINT_TYPE
+    )
+    X509_V_ERR_UNSUPPORTED_CONSTRAINT_SYNTAX = (
+        _lib.X509_V_ERR_UNSUPPORTED_CONSTRAINT_SYNTAX
+    )
+    X509_V_ERR_UNSUPPORTED_NAME_SYNTAX = (
+        _lib.X509_V_ERR_UNSUPPORTED_NAME_SYNTAX
+    )
+    X509_V_ERR_CRL_PATH_VALIDATION_ERROR = (
+        _lib.X509_V_ERR_CRL_PATH_VALIDATION_ERROR
+    )
+    X509_V_ERR_HOSTNAME_MISMATCH = _lib.X509_V_ERR_HOSTNAME_MISMATCH
+    X509_V_ERR_EMAIL_MISMATCH = _lib.X509_V_ERR_EMAIL_MISMATCH
+    X509_V_ERR_IP_ADDRESS_MISMATCH = _lib.X509_V_ERR_IP_ADDRESS_MISMATCH
+    X509_V_ERR_APPLICATION_VERIFICATION = (
+        _lib.X509_V_ERR_APPLICATION_VERIFICATION
+    )
+for name in list(globals().keys()):
+    if name.startswith("X509_V_"):
+        __all__.append(name)
```

Comment:
Instead of dumping everything at the top level, let's namespace it ala: https://github.com/pyca/pyopenssl/blob/main/src/OpenSSL/crypto.py#L1623